PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
```

Comment:
Python2 is won't be supported after this year. I'd use the print function `print('Getting auth token')` as you do below.